### PR TITLE
Add ETH Beijing to community events

### PR DESCRIPTION
**Pull request by AmazingAng**
## Description
Add ETH Beijing Hackathon
Details: https://github.com/WTFAcademy/ETHBeijing
Website (under construction): https://ethbeijing.com

## Related Issue
Add events to https://ethereum.org/en/community/events/

### CODE REVIEW

**Review by minimalsm — COMMENTED** [review]
Hey @AmazingAng, until we get a working/live/complete website, we won't be able to merge this. Please let us know when this is the case.

**Comment by AmazingAng** [comment]
> Hey @AmazingAng, until we get a working/live/complete website, we won't be able to merge this. Please let us know when this is the case.

Now the website is live: https://ethbeijing.xyz

File: src/data/community-events.json [hunk 1]
```diff
@@ -35,6 +35,15 @@
     "startDate": "2023-03-10",
     "endDate": "2023-03-29"
   },
+  {
+    "title": "ETH Beijing",
+    "to": "https://ethbeijing.com/",
```
Comment (corwintines) [inline src/data/community-events.json:40]:
```suggestion
    "to": "https://ethbeijing.xyz/",
```

Updated based on comment

**Review by corwintines — APPROVED** [review]

**Comment by corwintines** [comment]
Thanks @AmazingAng!

**Comment by corwintines** [comment]
@all-contributors can you add @AmazingAng for content

**Pull request merged**